### PR TITLE
Add psdk_ros2 source to Humble

### PR DESCRIPTION
**Pull request by vicmassy**
# Please Add This Package to be indexed in the rosdistro.

- psdk_ros2

# The source is here:

https://github.com/umdlife/psdk_ros2

# Checks
 - [x] All packages have a declared license in the package.xml
 - [x] This repository has a LICENSE file
 - [x] This package is expected to build on the submitted rosdistro

### CODE REVIEW

File: humble/distribution.yaml [hunk 1]
```diff
@@ -5235,6 +5235,16 @@ repositories:
       url: https://github.com/Simple-Robotics/proxsuite.git
       version: devel
     status: developed
+  psdk_ros2:
+    doc:
+      type: git
+      url: https://github.com/umdlife/psdk_ros2
```
Comment (mjcarroll) [inline humble/distribution.yaml:5241]:
```suggestion
      url: https://github.com/umdlife/psdk_ros2.git
```

File: humble/distribution.yaml [hunk 2]
```diff
@@ -5235,6 +5235,16 @@ repositories:
...
+      version: main
+    source:
+      type: git
+      url: https://github.com/umdlife/psdk_ros2
```
Comment (mjcarroll) [inline humble/distribution.yaml:5245]:
```suggestion
      url: https://github.com/umdlife/psdk_ros2.git
```

**Review by mjcarroll — COMMENTED** [review]
To make the linter happy

File: humble/distribution.yaml (hunk 2 above)
Comment (vicmassy) [reply humble/distribution.yaml]: Resolved in https://github.com/ros/rosdistro/pull/39574/commits/81e1e3e30076363e5c459ab6bcead4407e6d3a08

File: humble/distribution.yaml (hunk 1 above)
Comment (vicmassy) [reply humble/distribution.yaml]: Resolved in https://github.com/ros/rosdistro/pull/39574/commits/81e1e3e30076363e5c459ab6bcead4407e6d3a08

**Comment by mjcarroll** [comment]
Can you merge master into your branch again?  It looks like you may have gotten a broken version.

**Comment by vicmassy** [comment]
Done

**Review by mjcarroll — APPROVED** [review]

**Pull request merged**